schematic/manifest/generator.py

```diff
@@ -589,6 +573,72 @@ def get_manifest(self, json_schema = None):
 
         return manifest_url
 
+    def get_manifest(self, dataset_id: str, sheet_url: bool):
```

Comment:
@sujaypatil96 I see you updated functions that previous called get_manifest() as needed to get_empty_manifest or get_manifest(<dataset_id>, <sheet_url>).However, I was wondering, do you think it would make sense to have default argument values and behavior for the new get_manifest(<dataset_id>, <sheet_url>) function: dataset_id = None, sheet_url = True. In this default case, we can just replicate the behavior of the old get_manifest: call get_empty_manifest(). That way we preserve backward compatibility and don't need to change the signature of existing function calls.

Reply:
Yes, we could do that as well. In that case, the `get_manifest()` signature would change to `get_manifest(<dataset_id> = None, <sheet_url> = None, <json_schema> = None)`.

In addition to all the handlers I've written now, we can have a "default" else/elif block which returns `get_empty_manifest()` when either only `<json_schema>` argument is provided or no arguments are provided.

This sounds reasonable, and I think we can implement the behaviour this way. Should I go ahead and issue a new commit that does this @milen-sage?

Reply:
Yes, all that sounds good. Just a question - why not have sheet_url = True by default?

Reply:
Hm..I'm not sure, I think it's okay to leave `sheet_url = None`. The reason is because if it's defaulting to `True`, then we probably should have a handler for the case when the user sets it to `False`. In our default case, we are just making a call to `get_empty_manifest()`, which in any case returns a URL only.

Reply:
Ok - sounds good; I will test on my end, but please test on your end as well (it will likely take you less time!) :) 
Reminder: work hours boundaries :) apologies for posting the comment late - I saw yours and decided to reply. Thanks!

Reply:
No worries at all. Yes, I've tested it with different datasets on Synapse (RNAseqAlignment, FamilyHistory, etc. some of which have manifests and some of which don't), and also used different values for the `sheet_url` argument.

I've mentioned how you can test the method in the first PR message, you can just feed different values to the `examples/manifest_generator.py` script, so it should be fairly quick and easy. And when you're ready after the test, feel free to approve the PR and I can merge.